packages/integrations/cloudflare/src/index.ts

```diff
@@ -205,8 +205,9 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				if (isModeDirectory) {
 					const functionsUrl = new URL('functions', _config.root);
 					await fs.promises.mkdir(functionsUrl, { recursive: true });
-					const directoryUrl = new URL('[[path]].js', functionsUrl);
-					await fs.promises.rename(finalBuildUrl, directoryUrl);
+
+					const directoryUrl = new URL('functions/[[path]].js', _config.root);
+					await fs.promises.rename(finalBuildUrl, directoryUrl.pathname);
```

Comment:
This change to pathname seems to cause errors on windows builds.

Reply:
Fixed and pushed, thanks 🙏🏻